modules/olm-pruning-index-image.adoc

```diff
@@ -123,10 +123,13 @@ $ opm index prune \
     -f {index-image-pullspec} \// <1>
     -p {package1},{package2},{package3} \// <2>
     -t <target_registry>:<port>/<namespace>/{index-image} <3>
+    -i registry.redhat.io/openshift4/ose-operator-registry/{index-image} 
 ----
 <1> Index to prune.
 <2> Comma-separated list of packages to keep.
 <3> Custom tag for new index image being built.
+<4> Use `-i` only with IBM Power Systems and IBM Z images.
```

Comment:
So then:

<4> Use `-i` only with IBM Power Systems and IBM Z images to specify a multi-arch index image, eg., `-i registry.redhat.io/openshift4/ose-operator-registry/{index-image}`

?

Reply:
Will it be mentioned somewhere else what {index-image} to actuallly use? I believe we can explicitly say to use -i registry.redhat.io/openshift4/ose-operator-registry:v4.6 for both P and Z.